src/contact.rs

```diff
@@ -586,8 +586,8 @@ impl Contact {
 
         let mut add_self = false;
         let mut ret = Vec::new();
-        let flag_verified_only = listflags_has(listflags, DC_GCL_VERIFIED_ONLY);
-        let flag_add_self = listflags_has(listflags, DC_GCL_ADD_SELF);
+        let flag_verified_only = (listflags & DC_GCL_VERIFIED_ONLY) == DC_GCL_VERIFIED_ONLY;
```

Comment:
at similar places, we use `(listflags & DC_GCL_VERIFIED_ONLY) != 0` - imo, this makes things more readable - and saver against copy+paste errors.

Reply:
Fixed. I just copied what `listflags_has` did.